draft-ietf-ppm-dap.md

```diff
@@ -1248,11 +1253,8 @@ as follows:
 1. Initialize VDAF preparation and initial outputs as described in
    {{input-share-prep}}.
 
-[[OPEN ISSUE: consider moving the helper report ID check into
-{{input-share-validation}}]]
```

Comment:
Reviewer note: I believe this pertains to checking for distinctness of report IDs in the request: If so, this check does not go in {{input-share-validation}}, since that section is really about about per-report checks.